src/MarginTrading.Backend.Core/Services/ICfdCalculatorService.cs

```diff
@@ -4,6 +4,9 @@ public interface ICfdCalculatorService
     {
         decimal GetQuoteRateForBaseAsset(string accountAssetId, string instrument, string legalEntity);
         decimal GetQuoteRateForQuoteAsset(string accountAssetId, string instrument, string legalEntity);
+        decimal GetFplRate(string accountAssetId, string instrumentId, string legalEntity, bool fplSign);
+        decimal GetSwapRate(string accountAssetId, string instrumentId, string legalEntity, bool swapSign);
+        decimal GetMarginRate(string accountAssetId, string instrumentId, string legalEntity);
```

Comment:
All this *Sign are difficult to understand... (

Reply:
The best way to describe it I found is bool metricIsPositive.
In case of Fpl it goes as: order.Volume * (order.ClosePrice - order.OpenPrice) > 0
In case of rate for Account it goes as operation direction: direction == OrderDirection.Buy
In case of swaps it goes as: volume * swapRate > 0

So I renamed it, added default value = true for margin and let it be.